README.md

```diff
@@ -168,305 +116,247 @@ Typical Java threads
 by anything that meets the required interface. The name of the format identified should
 have parsers that meet the SessionParser interface configured into the factory.
 
 Once the container is identified, items are parsed out of it, built into IBaseDataObject instances
 by the SessionParser appropriate for that data type, and handed over to MobileAgent instances
-for processing as they become available from the pool. It is important to have in mind that the
+for processing as they become available from the pool. 
+
+It is important to have in mind that the
 Parser framework is operating on the single-threaded side of the system and handing items over to the
-multi-threaded (MobileAgent) side of the system. This implies that the parsers are fast enough
+multi-threaded (MobileAgent) side of the system. This necessitates that the parsers are fast enough
 to keep the number of configured threads busy.
 
 ## Development
 
 ### Requirements
 
-#### Linux or MacOSX operating system
-
-Currently, developing and running natively on Windows is not
-supported.  Use a virtualized Linux OS if you are using Windows.  
-
-In terms of Linux distributions, the two most widely used on the Emissary
-codebase are CentOS 6.X and Ubuntu 12.X.
-
 #### Java 1.8
 
-Java 1.6 and Java 1.7 are no longer supported in this codebase only 
-[Java 1.8](http://www.oracle.com/technetwork/java/javase/downloads/index-jsp-138363.html).  
-OpenJDK should work, although most testing is done with Oracle's JDK.  Be sure to use a recent version of
+Emissary requires [Java 1.8](http://www.oracle.com/technetwork/java/javase/downloads/index-jsp-138363.html).  
+Emissary now supports both OpenJDK and Oracle JDK.  Be sure to use a recent version of
 1.8 as there are bugs in earlier versions.
 
 #### Apache Maven 3.2
 
 Download and install [Apache Maven 3.2](http://maven.apache.org).
-Some maven commands are shown below.  A great place to start with Maven is the
-[5 min tutorial](https://maven.apache.org/guides/getting-started/maven-in-five-minutes.html).
-Another great resource explains the
-[lifecycles](https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html).
-We cover [configuring maven](#configuring-maven) below with Emissary
-specific settings.
-
-As you go through this guide, keep in mind the steps in the [default lifecycle](https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html#Lifecycle_Reference)
-are cumulative.  If you run *test*, Maven will run *compile* among other
-steps first.  Similarly, when you run *package*, Maven will run
-*compile* and *test* along with other steps first.
-
-We also use [maven profiles](http://maven.apache.org/guides/introduction/introduction-to-profiles.html).
-To activate specific functionality.  To activate profile abc, you would add `-Pabc`.  Mutliple profiles can be
-activated at once if you separate them with a comma.  Something like `-Pprofile-1,profile-2`. 
-
-#### Git 1.8+
```

Comment:
User can choose what version of Git they want and if they are on Github, we can treat that as assumed knowledge

Reply:
Version isn't required, but this info seems no less helpful than the other info.